torch/_dynamo/polyfills/itertools.py

```diff
@@ -68,7 +68,7 @@ def islice(iterable: Iterable[_T], /, *args: int | None) -> Iterator[_T]:
 # Reference: https://docs.python.org/3/library/itertools.html#itertools.pairwise
 if sys.version_info >= (3, 10):
     # itertools.pairwise was added at Python 3.10
-    @substitute_in_graph(itertools.pairwise, is_embedded_type=True)  # type: ignore[arg-type]
+    @substitute_in_graph(itertools.pairwise, is_embedded_type=True)  # type: ignore
```

Comment:
Please revert this and put the ignore code at the exact line where it occurs. Otherwise, it will disable type-checking in the whole function.